modules/integration_azure-mariadb/conf/00-heartbeat.yaml

```diff
@@ -0,0 +1,13 @@
+module: Azure MariaDB
+name: heartbeat
+
+transformation: true
+aggregation: ".mean(by=['azure_resource_id'])"
```

Comment:
I am afraid this id could change too often and cause undesirable alerts.
using same aggregation than other detectors could make this heartbeat less sensible to user change ? perhaps there is no possible change on it or you really want to be alerted of this kind of change, just a remark.
in any case using the id is not the most understandable when you receive an alert, in this case, do not use aggregation at all should provide same alerting behavior but the user will get all information instead of the id only.